src/components/hero.js

```diff
@@ -1,114 +1,251 @@
-import React from "react"
+import React, { useEffect, useRef, useState } from "react"
```

Comment:
I think it's for the best if you just separated the whole component. Don't clutter `hero.js`.
This would make unnecessary helper functions to be more isolated. It's harder to read.

Reply:
Good point, I'll break it up!

Reply:
Done here: https://github.com/reactph/reactjsph-website/pull/58/commits/94611073cf6840332ce4420706f91f65d3ef42c8.